<launch>

  <param name="/use_sim_time" value="true"/> 

  <env name="ROSCONSOLE_CONFIG_FILE" 
    value="$(find pandora_alert_handler)/config/alert_handler/logger_debug.cfg"/>
  <include file="$(find pandora_data_fusion_launchers)/launch/data_fusion_stand_alone.launch" /> 
  <include file="$(find pandora_vision_hole_detector)/launch/pandora_vision_hole_detector.launch" />

  <!--<node pkg="node_tests" type="bag_player_for_tests" name="player" -->
  <!--  args= />                                                       -->
  <!--<node pkg="rosbag" type="play" name="player" output="screen" args= />-->
  <!--<param name="test/subscribed_topic_names/holeDirection" type="str" value="/vision/hole_direction_alert" />-->
  <remap from="/kinect/depth_registered/points" to="/camera/depth_registered/points" />
  
  <node name="state_changer" type="state_changer" pkg="state_manager" args="2" />

  <node name="data_fusion_object_visualization" type="data_fusion_object_visualization"
    pkg="pandora_visualization" respawn="true" ns="data_fusion" />        

  <include file="$(find pandora_visualization)/launch/rviz/rviz.launch" /> 
  <node name="data_fusion_object_visualization" type="rqt_gui" pkg="rqt_gui" />
  <!--<test test-name="VisionDataFusionTest" pkg="pandora_alert_handler"-->
    <!--type="vision_data_fusion_test.py" />-->

</launch>
